src/services/requests.ts

```diff
@@ -11,8 +12,15 @@ import {
 
 /** Blog Posts */
 
-export function getBlogPostList(): Promise<ResponseBody<Array<PostShort>>> {
-  return request.get({ path: '/admin/blog/posts' });
+export function getBlogPostList(
+  requestParams?: TableDataRequestParams
+): Promise<ResponseBody<Array<PostShort>>> {
+  const baseUrl = '/admin/blog/posts';
+  const path = requestParams?.searchQuery
+    ? `${baseUrl}?query=${requestParams?.searchQuery}`
+    : baseUrl;
+
+  return request.get({ path });
```

Comment:
please investigate what params can be passed into `request.get` and make this request simpler